### ISSUE TITLE
Build failure on v0.20

### ISSUE THREAD

**Issue by PassiveLemon** [WiVRn/WiVRn#154]
```
/nix/store/vcvhwiilizhijk7ywyn58p9l005n9sbn-binutils-2.43.1/bin/ld: CMakeFiles/wivrn-dashboard.dir/wivrn-dashboard_autogen/GGJYZH3U72/qrc_wivrn-dashboard_translations.cpp.o: in function `qInitResources_wivrn_dashboard_translations()':
qrc_wivrn-dashboard_translations.cpp:(.text+0x0): multiple definition of `qInitResources_wivrn_dashboard_translations()'; CMakeFiles/wivrn-dashboard.dir/__/.qt/rcc/qrc_wivrn-dashboard_translations.cpp.o:qrc_wivrn-dashboard_translations.>
/nix/store/vcvhwiilizhijk7ywyn58p9l005n9sbn-binutils-2.43.1/bin/ld: CMakeFiles/wivrn-dashboard.dir/wivrn-dashboard_autogen/GGJYZH3U72/qrc_wivrn-dashboard_translations.cpp.o: in function `qCleanupResources_wivrn_dashboard_translations()':
qrc_wivrn-dashboard_translations.cpp:(.text+0x40): multiple definition of `qCleanupResources_wivrn_dashboard_translations()'; CMakeFiles/wivrn-dashboard.dir/__/.qt/rcc/qrc_wivrn-dashboard_translations.cpp.o:qrc_wivrn-dashboard_translati>
collect2: error: ld returned 1 exit status
make[2]: *** [dashboard/CMakeFiles/wivrn-dashboard.dir/build.make:334: server/wivrn-dashboard] Error 1
make[1]: *** [CMakeFiles/Makefile2:5166: dashboard/CMakeFiles/wivrn-dashboard.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```
I have a slight feeling this might be a Nix problem. I built and was using v0.20 for some time before this issue appeared.

Building with the dashboard disabled causes it to go away.

**Comment by Meumeu** [comment]
I've had the same issue on arch, I think it's because of qt 6.8. I'll put the fix in a branch soon.

**Comment by Meumeu** [comment]
It's fixed in master since d85916b6aaba3b4e9a7367e973b14f60507701fd

**Comment by PassiveLemon** [comment]
How long would you estimate until this gets released?

**Comment by Meumeu** [comment]
Fixed in 0.21
